### PR TITLE
[ST-828] doc for robot plugin

### PR DESCRIPTION
**Pull request by Konboi**
SSIA

### CODE REVIEW

File: docs/optimizing-test-execution/subsetting-tests.md [hunk 1]
```diff
@@ -210,6 +211,24 @@ The file will contain the subset of test that should be run. You can now invoke
...
+
+```bash
+robot --dryrun -o dryrun.xml
+launchable subset \
+  --build <build name> \
+  --target <target> \
+  robot dryrun.xml > launchable-subset.txt
+```
+
+the file will contain the subset of test that should be run. you can now invoke minitest to run exactly those tests:
```
Comment (awilkes) [inline docs/optimizing-test-execution/subsetting-tests.md:226]:
```suggestion
The file will contain the subset of test that should be run. You can now invoke Robot to run exactly those tests:
```

File: docs/optimizing-test-execution/subsetting-tests.md [hunk 2]
```diff
@@ -210,6 +211,24 @@ The file will contain the subset of test that should be run. You can now invoke
 bundle exec rails test $(cat launchable-subset.txt)
 ```
 
+### Robot
+
+To select meaningful subset of tests, use `--dryrun` option to get all test cases, like this:
```
Comment (awilkes) [inline docs/optimizing-test-execution/subsetting-tests.md:216]:
```suggestion
To select a meaningful subset of tests, use `--dryrun` option to list all test cases, like this:
```

**Review by awilkes — APPROVED** [review]

**Comment by Konboi** [comment]
Thank you for your review!

File: docs/optimizing-test-execution/subsetting-tests.md [hunk 3]
```diff
@@ -210,6 +211,24 @@ The file will contain the subset of test that should be run. You can now invoke
...
+launchable subset \
+  --build <build name> \
+  --target <target> \
+  robot dryrun.xml > launchable-subset.txt
+```
+
+The file will contain the subset of test that should be run. You can now invoke Robot to run exactly those tests:
+
+```bash
+bundle exec rails test $(cat launchable-subset.txt)
```
Comment (awilkes) [inline docs/optimizing-test-execution/subsetting-tests.md:229]: Oh @Konboi what's the right `robot` command here?

Comment (Konboi) [reply docs/optimizing-test-execution/subsetting-tests.md:229]: yes right, just moment

Comment (awilkes) [reply docs/optimizing-test-execution/subsetting-tests.md:229]: I think it may be `robot $(cat launchable-subset.txt)`.

Comment (Konboi) [reply docs/optimizing-test-execution/subsetting-tests.md:229]: I thought that but I ran it and failed, so I'll investigate it. 

```
$ robot . $(cat subset.txt) .
[ ERROR ] Parsing '-s' failed: File or directory to execute does not exist.

Try --help for usage information.
```

Comment (Konboi) [reply docs/optimizing-test-execution/subsetting-tests.md:229]: `robot $(cat launchable-subset.txt) .` is correct.
The cause of the error was dependent on my environment.

Comment (Konboi) [reply docs/optimizing-test-execution/subsetting-tests.md:229]: :pray:
https://github.com/launchableinc/cli/pull/174

**Pull request merged**